src/stories/ButtonWithDropdown.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ButtonWithDropdown from '../components/ButtonWithDropdown';
+
+/**
+ * We use the Component Story Format for writing stories. Follow the docs here:
+ *
+ * https://storybook.js.org/docs/react/writing-stories/introduction#component-story-format
+ */
```

Comment:
This is cool, love the accessibility warnings built into the tool. It would be great to share a brief email/message about the tool being introduced to the project and best practices for using it.

Reply:
What I don't quite understand is if this is bi-directional. I can see that the code defined in the .stories file defines the generated UI/preview. But can we actually use the defined templates in our Components? 

For example, the Button has Default and Loading args, but I'm not able to find a reference to this in code.

Reply:
>It would be great to share a brief email/message about the tool being introduced to the project and best practices for using it.

That's a good idea, I think I'll work on this today thanks!

>I can see that the code defined in the .stories file defines the generated UI/preview. But can we actually use the defined templates in our Components?

No, they are only used to generate the "examples" of the components in the storybook docs.

Reply:
Nice, just seen the PR 👍